cubic-server/interface/LogsInterface.cpp

```diff
@@ -4,23 +4,64 @@
 #include <queue>
 #include <unistd.h>
 
-#include "logging/Logger.hpp"
 #include "LogsInterface.hpp"
 
 LogsInterface::LogsInterface() :
-  m_VBox(Gtk::ORIENTATION_VERTICAL)
+  m_VBox_main(Gtk::ORIENTATION_VERTICAL),
+  m_VBox_logs(Gtk::ORIENTATION_VERTICAL),
+  m_HBox_filters(Gtk::ORIENTATION_HORIZONTAL),
+  m_Info("Info"),
+  m_Warn("Warning"),
+  m_Error("Error"),
+  m_Fatal("Fatal"),
+  m_Debug("Debug"),
+  m_Reset("↺")
```

Comment:
no sure about this character for reseting filters but I'm not a designer so I will not say anything ^^